opm/parser/eclipse/EclipseState/EclipseState.cpp

```diff
@@ -99,6 +97,9 @@ namespace Opm {
 
     
     MessageContainer& EclipseState::getMessageContainer() {
+        m_messageContainer.appendMessages(m_tables.getMessageContainer());
```

Comment:
This interface is broken. 

If you call `getMessageContainer` (which is named and formed as a read-as-is) multiple times you will likely just collect the set of sub container messages as many times.